bears/java/CheckstyleBear.py

```diff
@@ -75,9 +82,13 @@ def create_arguments(
         check_invalid_configuration(
             checkstyle_configs, use_spaces, indent_size)
 
-        if checkstyle_configs in known_checkstyles:
+        if checkstyle_configs in _checkstyle_provided_styles:
```

Comment:
We can expect future jar to contain these ruleset XML files.
I assume they test against these rulesets, so we should trust them.
If they disappear, we'll find another solution.